builders/get-started/moonbeam-dev.md

```diff
@@ -166,9 +166,11 @@ When running a node via the binary file, data is stored in a local directory typ
 ./target/release/moonbeam purge-chain --dev -y
 ```
 
-This will remove the data folder, note that all chain data is now lost.
+This will remove the data folder, note that all chain data is now lost. To learn more about all of the available `purge-chain` commands, you can checkout the [Purging Binary Data](/node-operators/networks/compile-binary/#purging-binary-data) section of our documentation.
+
+!!! note
+    If you used Docker, the data folder is related to the Docker container itself.
```

Comment:
I don't think this is entirely true, when I run my Moonbase Alpha node I specify a volume where the data is stored:

```
-v "/var/lib/alphanet-data:/data"
```

Not sure what is the default if this is not provided

Reply:
> So what about if I'm a node operator and I want to purge my chain?

I'm not sure what you mean by this? What is the alternative to being a node operator?

> In that code snippet I do see the stuff about Moonbase Alpha will be ocasionally purged (which does not seem to be the case). Can you change that part?

What code snippet are you talking about?

Reply:
I didn't realize you could specify chain data using Docker - I'll update the docs for the Docker section so people know they can do that

Reply:
When running the docker command you can specify where the data is stored, I'm not sure what happens when you don't specify anything.

Reply:
this is outdated, thats been fixed 😄

Reply:
just about to push up the changes for the reorg but I like to review it in the morning now so I dont make little mistakes that can be easily avoided with a fresh mind haha